examples/hosts/list_all_group_member_ids.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+"""Caracara Examples Collection.
+
+list_all_groups_members_ids.py
```

Comment:
I think this file just does the same thing as `list_all_group_members.py` now?

Reply:
I think the concept of returning the ID list needs to be revisted in general.  To your point, the combined endpoint solution returns us this detail, and it gives it back faster.  The only question is if there are pagination differences, but I'm pretty sure those are solved by the batching.

Reply:
Gotcha. Are we seeing any differences in the actual data returned? If not, we should provide one way (the best way) to do this, and make this obvious to users. There are a _mountain_ of functions, as it happens, for interfacing with host groups, including lots of `get_` and `describe_` functions. Can we try to work to reduce those down so that the way to do things is more obvious? Once we do that, hopefully one of these examples will naturally fall away.